src/improvements/template/EnableDeputyPauseModuleTemplate.sol

```diff
@@ -11,6 +11,10 @@ import {SimpleTaskBase} from "src/improvements/tasks/types/SimpleTaskBase.sol";
 import {ModuleManager} from "lib/safe-contracts/contracts/base/ModuleManager.sol";
 import {AccountAccessParser} from "src/libraries/AccountAccessParser.sol";
 
+interface Isafe {
```

Comment:
Unused?

Reply:
Yeah both I was removing this at the moment actually let me push this right now

Reply:
65f50e38c5b12b92f7af0975b14f86d0900624f9